client/gutenberg/editor/controller.js

```diff
@@ -186,6 +187,17 @@ export const authenticate = ( context, next ) => {
 		`${ siteAdminUrl }../wp-login.php`
 	);
 
+	if ( siteId === 204039232 ) {
+		logToLogstash( {
+			feature: 'calypso_client',
+			message: 'e2e atomic auth redirect',
+			severity: 'debug',
+			extra: {
+				state: JSON.stringify( state, null, 2 ),
+			},
+		} );
+	}
```

Comment:
Let's add a comment before shipping. `TODO: Remove me ASAP` or something does the job.

Reply:
It looks like it won't be necessary to merge it, so I'll just close the PR when ready. Does that sound OK?